enterprise/frontend/src/metabase-enterprise/auth/components/SettingsJWTForm/SettingsJWTForm.unit.spec.tsx

```diff
@@ -0,0 +1,185 @@
+import fetchMock from "fetch-mock";
+
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen, waitFor } from "__support__/ui";
+import { createMockGroup } from "metabase-types/api/mocks";
+
+import { SettingsJWTForm } from "./SettingsJWTForm";
+
+const GROUPS = [
+  createMockGroup(),
+  createMockGroup({ id: 2, name: "Administrators" }),
+  createMockGroup({ id: 3, name: "foo" }),
+  createMockGroup({ id: 4, name: "bar" }),
+  createMockGroup({ id: 5, name: "flamingos" }),
+];
+
+const elements = [
```

Comment:
It'd be cool if we could derive this from the settings exported in `admin/selectors`, but it's probably not really worth spending time on. (unless you've developed a love for admin settings and forms haha)